controllers/user.controller.js

```diff
@@ -0,0 +1,70 @@
+const User=require("../models/user-model");
+
+const getUsers=async(req,res)=>{
+  try{
+    let users=await User.find({})
+    users=users.map((user)=>{
+      user.password=undefined;
+      return user;
+    })
+    res.json({success:true , users})
+  }catch(err){
+    console.log(err.message)
+    res.status(500).json({
+      success:false,
+      message:"Unable to get Users list",
+      errMessage:err.message
+    })
+  }
+}
+
+const findUser=async(req,res)=>{
+  const {email,password}=req.body;
+  const userExists=await User.exists({email})
+  if(userExists){
+    let user=await User.findOne({ email , password })
+    if(user){
```

Comment:
It's better to send the email and password in the headers. Then, here you would not need a POST  req on "/login" route.
You can instead do a GET req on "/login" route and read email ,password from headers

Reply:
Yeah we can use that too but I think we can do through this also

Reply:
Sorry, the POST req on "/login" is right .  But I have heard that its a good practice to send email and password in headers. Please verify it !